wait-for-plugins.sh

```diff
@@ -3,17 +3,26 @@
 # This script is a workaround for podman-compose absence of support for depends_on
 
 DYNAMIC_PLUGINS_CONFIG="dynamic-plugins-root/app-config.dynamic-plugins.yaml"
+USER_APP_CONFIG="configs/app-config/app-config.local.yaml"
+DEFAULT_APP_CONFIG="configs/app-config/app-config.yaml"
 
-
+# Wait for dynamic plugins config to be generated
 while [ ! -f "$DYNAMIC_PLUGINS_CONFIG" ]; do
     echo "Waiting for $DYNAMIC_PLUGINS_CONFIG to be created by install-dynamic-plugins container ..."
-    sleep 2 
+    sleep 2
 done
 
+# Optionally include user app-config.local.yaml if it exists
+EXTRA_CLI_ARGS=""
+if [ -f "$USER_APP_CONFIG" ]; then
+    echo "Using user app-config.local.yaml"
+    EXTRA_CLI_ARGS="--config $USER_APP_CONFIG"
+fi
 
+# Run Backstage with default + optional config overrides
 node packages/backend --no-node-snapshot \
-    --config app-config.yaml \
+    --config "$DEFAULT_APP_CONFIG" \
```

Comment:
```suggestion
    --config app-config.yaml \
```

The order of the app-config files matters.
This is a file present in the image, and it needs to be first in the list.